src/bt_deployer.js

```diff
@@ -173,19 +179,54 @@ class BTDeployer {
     logger.info(`originGateway address ${originGateway.address}`);
     logger.info(`auxiliaryCoGateway address ${auxiliaryCoGateway.address}`);
 
+
+    const brandedToken = new ContractInteract.BrandedToken(
+      this.origin.web3,
+      this.chainConfig.brandedToken.address,
+    );
+
+    const liftRestrictionTxOptions = {
+      gasPrice: this.origin.txOptions.gasPrice,
+      from: this.origin.masterKey,
+    };
+    const stakeVaultAddress = await originGateway.getStakeVault();
+
+    await brandedToken.liftRestriction(
```

Comment:
Maybe this should happen in setup? 
Should we create a ticket?

Reply:
Yes, we should have it in setup.
Created ticket :-
`https://github.com/OpenST/brandedtoken.js/issues/125`

Reply:
Thank you 👍